src/index.js

```diff
@@ -88,7 +100,7 @@ class ExtractTextPlugin {
     if (Array.isArray(options) || isString(options) || typeof options.options === 'object' || typeof options.query === 'object') {
       options = { use: options };
     } else {
-      validateOptions(path.resolve(__dirname, '../schema/loader.json'), options, 'Extract Text Plugin (Loader)');
+      validateOptions(path.resolve(__dirname, './schema/loader.json'), options, 'Extract Text Plugin (Loader)');
```

Comment:
Should also be reverted.

Reply:
If I run the tests with ../schema I get:

    ENOENT: no such file or directory, open '/home/david/Projekte/extract-text-webpack-plugin/schema/loader.json'

Reply:
Because you moved the schema into the src folder, but @michael-ciniawsky wants to have them in a different location anyway, see comment.